src/Pages/License/ModifyProductKey/index.jsx

```diff
@@ -0,0 +1,14 @@
+import Content from "Layouts/Content/Content";
+import ProductKey from "./ProductKey";
+import LicenseDetails from "../Details";
+
+const ModifyProductKeyWrapper = () => {
```

Comment:
rename to ModifyLicenseWrapper.
Also, rename folder to just 'Modify'

Reply:
done